File: app/src/main/java/com/univreview/model/Term.java
package com.univreview.model;

import java.io.Serializable;

/**
 * Created by DavidHa on 2017. 3. 11..
 */
public class Term implements Serializable {
    public String startDate;
    public String endDate;

    @Override
    public String toString() {
        return "Term{" +
                "startDate='" + startDate + '\'' +
                ", endDate='" + endDate + '\'' +
                '}';
    }
}

File: app/src/main/java/com/univreview/model/model_kotlin/Term.kt
package com.univreview.model.model_kotlin

import com.google.gson.annotations.SerializedName

/**
 * Created by DavidHa on 2017. 9. 1..
 */
data class Term(@SerializedName("started_at") val startedAt: String,
                @SerializedName("ended_at") val endedAt: String)